src/frontend/src/components/loader.ts

```diff
@@ -1,13 +1,20 @@
 import { ERROR_SUPPORT_URL } from "$src/config";
 import { html, render } from "lit-html";
-import loaderUrl from "./loader.svg";
+
+// Use same import approach as in 'src/frontend/src/flows/dappsExplorer/dapps.ts'
```

Comment:
Ah, nice, maybe we can do the same in the other and it fixes the issue with the types?

Reply:
This doesn't fix the issue with types, it fixes the issue with the actual imported data being inconsistent between showcase and build.

Reply:
thanks for the explanation!